portfolio/src/main/webapp/index.html

```diff
@@ -33,54 +33,108 @@
 					<span class="link-text">Experience</span>
 				</a>
 			</li>
+			<div class="color-buttons">
```

Comment:
This is all four spaces again. I think the cloud shell editor goes back to 4 whenever you reload it - I set it to 2 permanently by going into File > Settings > Open Preferences > Editor and changing tabSize.

Reply:
Thanks! Yeah I couldn't figure out how to make it permanently two spaces